schema/column.go

```diff
@@ -92,6 +97,7 @@ func (c Column) ToArrowField() arrow.Field {
 	if c.PrimaryKeyComponent {
 		mdKV[MetadataPrimaryKeyComponent] = MetadataTrue
 	}
+	mdKV[MetadataTypeSchema] = c.TypeSchema
```

Comment:
We need to put the schema in the field metadata for the sake of the `cloudquery tables` command. Tables and fields get serialized over the wire (during the gRPC call), so if we want the `type_schema` to appear in the result of the tables command we need to send it via metadata

Reply:
No reason, fixed in https://github.com/cloudquery/plugin-sdk/pull/1796/commits/d4044fc5ecc54531397808d87f1e3c13f19e3488